src/components/Reviews.jsx

```diff
@@ -30,17 +43,57 @@ export default function Reviews() {
     );
   }
 
+  const handleChange = (event) => {
+    setSortType(event.target.value);
+  };
+
+  const handleRadioChange = (event) => {
+    setSortOrder(event.target.value);
+  };
+
   return (
     <Box id="all-page" sx={{ width: "100%" }}>
       <header id="all-reviews">
-        <h2 >All Reviews</h2>
+        <h2>All Reviews</h2>
         <Tooltip id="filter-tip" title="Click here to filter by category.">
           <Link to="/api/categories">
             <IconButton>
               <FilterListIcon />
             </IconButton>
           </Link>
         </Tooltip>
+
+        <Box sx={{ minWidth: 120 }}>
+          <FormControl fullWidth>
```

Comment:
 We want to make sure that reviews are sorted by default to **date** **descending** - so we may want the drop down to default to those upon loading the page.